grouper/permissions.py

```diff
@@ -387,6 +491,15 @@ class RequestAlreadyGranted(PermissionRequestException):
     """Group already has requested permission + argument pair."""
 
 
+# hmm maybe don't need this. people can do things to the permission,
+# e.g., grant it to groups, request grants for it, revoke it from
+# groups, etc. and we kind of don't care, as long as the permission's
+# disabled state prevents it from being used, which is the important
+# bit
```

Comment:
Yea, my main worry right now is it seems pretty easy to accidentally use a disabled perm in future code, e.g. by seeing that there are a bunch of `session.query`s scattered around and writing one that forgets the enabled filter. Don't think using this exception would fix that, and not sure how to fix it in general :/ Maybe some way to eliminate all these `session.query`s would help? Not sure how to do that though